pwndbg/commands/probeleak.py

```diff
@@ -71,30 +73,54 @@ def flags_str2int(flags_s):
     help="Mapping name of the page that you want the pointers point to",
 )
 parser.add_argument(
-    "--max-ptrs", type=int, default=0, help="Stop search after find n pointers, default 0"
+    "--max-ptrs",
+    type=int,
+    default=0,
+    help="Stop search after find n pointers, default 0",
 )
 parser.add_argument(
     "--flags",
     type=str,
     default=None,
     help="flags of the page that you want the pointers point to. [e.g. rwx]",
 )
+parser.add_argument(
+    "--permissions",
+    type=str,
+    default=None,
+    help="Search for pointers to pages with specific permissions [e.g., rwx]",
+)
+
+parser.add_argument(
+    "--page-name",
+    type=str,
+    default=None,
+    help="Search for pointers to pages with a specific name",
+)
```

Comment:
The `--page-name` does the same thing as `--point-to`. We should have only one argument like this.

The description `--point-to` should probably be fixed